rules/ensure-system-master-group-is-not-set/rule.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "ensure-system-master-group-is-not-set",
+    "attributes": {
+      "armoBuiltin": true,
+      "actionRequired": "manual review",
+      "hostSensorRule": false,
+      "useFromKubescapeVersion": "",
+      "useUntilKubescapeVersion": "",
+      "imageScanRelated": false,
+      "controlConfigInputs": ""
+    },
+    "ruleLanguage": "Rego",
+    "match": [
+        {
+          "apiGroups": [
+            "rbac.authorization.k8s.io"
+          ],
+          "apiVersions": [
+            "v1"
+          ],
+          "resources": [
+            "ClusterRoleBinding",
+            "RoleBinding"
+          ]
+        }
+    ],
```

Comment:
do we need to request these resources?

Reply:
no more, I'm going to remove them. thanks :)